app/controllers/grant_submissions/submissions/submission_applicants_controller.rb

```diff
@@ -21,6 +21,7 @@ def create
         else
           submission_applicant = GrantSubmission::SubmissionApplicant.create(submission: @submission, applicant: user)
           if submission_applicant.errors.none?
+            ApplicantMailer.assignment(submission_applicant: submission_applicant).deliver_now
             flash[:success] = "#{helpers.full_name(submission_applicant.applicant)} was added as applicant on #{@submission.title}."
```

Comment:
Wording seems a bit awkward. "...added as an to" or "...is now an applicant" or "You added [name] as an applicant"... or something else?